job-server-tests/src/main/java/spark.jobserver/JavaHelloWorldJob.java

```diff
@@ -1,12 +1,12 @@
-package spark.jobserver;
+// package spark.jobserver;
```

Comment:
It doesn’t compile  :(   I can’t figure out why.     I’m hoping to create a new Java-specific API that doesn’t inherit from SparkJobBase, it will be much easier.

> On Jun 1, 2016, at 11:17 AM, Fernando Otero <notifications@github.com> wrote:
> 
> In job-server-tests/src/main/java/spark.jobserver/JavaHelloWorldJob.java <https://github.com/spark-jobserver/spark-jobserver/pull/442#discussion_r65414535>:
> 
> > @@ -1,12 +1,12 @@
> > -package spark.jobserver;
> > +// package spark.jobserver;
> are we removing this whole class?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/spark-jobserver/spark-jobserver/pull/442/files/6c9e4d963c90a8f9ce50b818440d3f2c16149699#r65414535>, or mute the thread <https://github.com/notifications/unsubscribe/ABA32yg3x6wABWXdfwggj4S8A8g24S_5ks5qHcyggaJpZM4IQq8l>.
>